src/monmove.cpp

```diff
@@ -271,6 +277,113 @@ float monster::rate_target( Creature &c, float best, bool smart ) const
     return INT_MAX;
 }
 
+std::map<item *, const tripoint> monster::find_loot_in_radius( const tripoint &target, int radius )
+{
+    bool steals_food = has_flag( MF_STEALS_FOOD );
+    bool steals_shiny = has_flag( MF_STEALS_SHINY );
+    bool suitable = false;
+    std::map<item *, const tripoint> lootable;
+
+    //Check all tiles nearby within radius
+    for( const tripoint &p : g->m.points_in_radius( target, radius ) ) {
+        //Check if we can see some items on the tile
```

Comment:
Please don't comment every line by just repeating what the code already says. This line for example is completely redundant. One can pretty much assume that a function called `sees_some_items( some_point, some_creature )` checks whether the given creature can see some items at the given point. No need to repeat this.